channel-sender/lib/channel_sender_ex/core/channel.ex

```diff
@@ -287,11 +297,6 @@ defmodule ChannelSenderEx.Core.Channel do
     {:stop, :normal, %{data | stop_cause: :name_conflict}}
   end
 
-  def connected(:info, _m = {:DOWN, _ref, :process, _object, _reason}, _data) do
```

Comment:
this is not used?

Reply:
This method signature its the same as the previous method, and it doesnt perform the necesary logic. So its safe to eliminate.